tests/Matcher/JsonMatcherTest.php

```diff
@@ -113,7 +112,7 @@ public function test_error_when_path_in_nested_value_does_not_exist()
 
         $this->assertFalse($this->matcher->match($value, $pattern));
 
-        $this->assertEquals($this->matcher->getError(), 'There is no element under path [foo][bar][faz] in value.');
+        $this->assertEquals($this->matcher->getError(), 'Value {"foo":{"bar":[]}} does not match pattern {"foo":{"bar":{"faz":"faz value"}}}');
```

Comment:
this potentially might look opposite to increasing verbosity but I'm planning to create matcher backtrace that will let all matchers to put each operation into the backtrace, currently in most cases errors from JSON/XML matcher does not make any sense except situations like this one (it is handled in special way).